config/config.go

```diff
@@ -508,6 +527,14 @@ func (cfg Config) Validate() error {
 		)
 	}
 
+	if cfg.Twilio.VoiceName != "" {
+		err = validate.Many(
+			err,
+			validate.ASCII("Twilio.VoiceName", cfg.Twilio.VoiceName, 1, 50),
+			validate.ASCII("Twilio.VoiceLanguage", cfg.Twilio.VoiceLanguage, 0, 10),
+		)
+	}
```

Comment:
We can simplify this since neither is required

```suggestion
	err = validate.Many(
		err,
		validate.ASCII("Twilio.VoiceName", cfg.Twilio.VoiceName, 0, 50),
		validate.ASCII("Twilio.VoiceLanguage", cfg.Twilio.VoiceLanguage, 0, 10),
	)
```

Reply:
adjusted as per your request.  Technically you require a language if you specify a voice, which I adjusted the check to validate.